deploy/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+heapster:
+    image: kubernetes/heapster:v0.14.3
+    ports:
+        - "8002:8002"
```

Comment:
Yes. I will fix this alone with adding a cadvisor service in the compose
file

Le vendredi 26 juin 2015, Vish Kannan <notifications@github.com> a écrit :

> In deploy/docker-compose/docker-compose.yml
> <https://github.com/GoogleCloudPlatform/heapster/pull/374#discussion_r33380496>
> :
>
> > @@ -0,0 +1,18 @@
> > +heapster:
> > +    image: kubernetes/heapster:v0.14.3
> > +    ports:
> > +        - "8002:8002"
>
> This should be 8082 I think.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/heapster/pull/374/files#r33380496>
> .
>